ams/dashboard/src/components/tables-sub-menu/TablesMenu.vue

```diff
@@ -271,6 +286,13 @@ export default defineComponent({
       if (!state.curCatalog || !state.database) {
         return
       }
+      if (state.allTableListLoaded.length) {
```

Comment:
I'm not sure if the tableList here is still old after switching database, if yes,  maybe you need to save a mapping from database to tableList?

Reply:
No, in the `const handleClickDb = (item: IDatabaseItem)  function`, i have set `state.allTableListLoaded.length = 0`.